scripts/create_db.sh

```diff
@@ -52,40 +52,40 @@ db_user="postgres"
 db_pw="docker"
```

Comment:
Please remove the commented out code. It is still in git and we can get it again if needed.